.github/workflows/openblas.yml

```diff
@@ -0,0 +1,144 @@
+
+      - name: Docker build
+        run: |
+          docker build --build-arg BASE_IMAGE=${{ env.BASE_IMAGE }} \
+                       --build-arg CMAKE_VER=${{ env.CMAKE_VER }} \
+                       --build-arg CCACHE_TAR_NAME=${{ env.CCACHE_TAR_NAME }} \
+                       -t open3d-openblas-ci:latest \
+                       -f .github/workflows/Dockerfile.openblas .
+
+      - name: C++ tests
+        run: |
+          docker run --rm open3d-openblas-ci:latest build/bin/tests
+
+      - name: Python tests
+        run: |
+          docker run --rm open3d-openblas-ci:latest \
+            pytest python/test/ --ignore python/test/ml_ops/ --ignore python/test/t/io/test_realsense.py
+
+      # https://stackoverflow.com/a/34093828/1255535
+      - name: Copy ccache out from the container
+        run: |
+          docker run --rm --entrypoint \
+            cat open3d-openblas-ci:latest /${{ env.CCACHE_TAR_NAME }}.tar.gz \
+            > ${GITHUB_WORKSPACE}/${{ env.CCACHE_TAR_NAME }}.tar.gz
+
+      - name: GCloud CLI setup
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          version: ${{ env.GCE_CLI_GHA_VERSION }}
+          service_account_key: ${{ secrets.GCE_SA_KEY_DOCS_CI }}
+          project_id: ${{ secrets.GCE_DOCS_PROJECT }}
+          export_default_credentials: true
+
+      - name: Upload ccache to GCS
+        run: |
+          gsutil cp ${GITHUB_WORKSPACE}/${{ env.CCACHE_TAR_NAME }}.tar.gz gs://open3d-ci-cache/ || true
+
+
+  openblas_arm64:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    env:
+      BASE_IMAGE: arm64v8/ubuntu:20.04
+      CMAKE_VER: cmake-3.19.7-Linux-aarch64
+      CCACHE_TAR_NAME: open3d-arm64-ci-ccache
+    steps:
+      - name: Cancel outdated
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          github_token: ${{ github.token }}
+
+      - name: Checkout source code
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Maximize build space
+        run: |
+          source util/ci_utils.sh
+          maximize_ubuntu_github_actions_build_space
+
+      # https://www.stereolabs.com/docs/docker/building-arm-container-on-x86/
+      - name: Setup Docker with QEMU
+        run: |
+          sudo apt-get --yes install qemu binfmt-support qemu-user-static
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+
+      - name: Docker build
+        run: |
+          docker build --build-arg BASE_IMAGE=${{ env.BASE_IMAGE }} \
+                       --build-arg CMAKE_VER=${{ env.CMAKE_VER }} \
+                       --build-arg CCACHE_TAR_NAME=${{ env.CCACHE_TAR_NAME }} \
+                       -t open3d-openblas-ci:latest \
+                       -f .github/workflows/Dockerfile.openblas .
+
+      - name: C++ tests
+        run: |
+          docker run --rm open3d-openblas-ci:latest build/bin/tests \
+            --gtest_filter="-*CreateFromRGBDImage*"
```

Comment:
`CreateFromRGBDImage` should work after PR #3207

Reply:
Done